Source/WebCore/platform/graphics/ca/cocoa/PlatformCAFiltersCocoa.mm

```diff
@@ -192,8 +192,8 @@ static unsigned keyValueCountForFilter(const FilterOperation& filterOperation)
             const auto& dropShadowOperation = downcast<DropShadowFilterOperation>(filterOperation);
             auto size = CGSizeMake(dropShadowOperation.x(), dropShadowOperation.y());
             [presentationModifiers[i].second.get() setValue:[NSValue value:&size withObjCType:@encode(CGSize)]];
-            [presentationModifiers[i++].second.get() setValue:(id) cachedCGColor(dropShadowOperation.color()).autorelease()];
-            [presentationModifiers[i++].second.get() setValue:@(dropShadowOperation.stdDeviation())];
+            [presentationModifiers[++i].second.get() setValue:(id) cachedCGColor(dropShadowOperation.color()).autorelease()];
+            [presentationModifiers[++i].second.get() setValue:@(dropShadowOperation.stdDeviation())];
```

Comment:
This is fairly subtle.

I understand you need to increment `i` to move through the modifiers, but maybe it would be clearer as

```
array[i].whatever();
array[i + 1].whatever();
array[i + 2].whatever();

i += 2; // advance past the color and deviation

Reply:
Will do that in the commit, thanks for the suggestion!